float64 x
float64 y
float64 z
float64 rotate_angle
float64 tilt_angle
float64 grasp_width_first
float64 grasp_width_second
